template/content/CarterTemplate.csproj

```diff
@@ -1,11 +1,11 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net7.0</TargetFramework>
     <AssemblyName>CarterTemplate</AssemblyName>
     <OutputType>Exe</OutputType>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Carter" Version="6.1.1" />
+    <ProjectReference Include="..\..\src\Carter\Carter.csproj" />
```

Comment:
Eh? :)

Reply:
I assume you're referring to the fact that we swap the package reference for a project reference?

If so, please look at [the "Consideration" section on the other PR](https://github.com/CarterCommunity/Carter/pull/302#:~:text=302.build.3260679089-,Considerations,-Today%2C%20the%20template).

Reply:
I assume you refer to the package reference now being a project reference?

If so, please have a look at [the "Considerations" section on the other PR](https://github.com/CarterCommunity/Carter/pull/302#:~:text=302.build.3260679089-,Considerations,-Today%2C%20the%20template).

It's being replaced at pack time for a package reference; hopefully the description of this PR gives enough details.